src/main/java/com/minecolonies/colony/buildings/BuildingFisherman.java

```diff
@@ -53,6 +56,20 @@ public void readFromNBT(NBTTagCompound compound)
         super.readFromNBT(compound);
     }
 
+    @Override
+    public void onUpgradeComplete(final int newLevel)
+    {
+        super.onUpgradeComplete(newLevel);
+
+        EntityPlayer owner = ServerUtils.getPlayerOnServerFromUUID(this.getColony().getPermissions().getOwner());
+
+        if (newLevel == 1) {
+            owner.triggerAchievement(ModAchievements.achBuildingFisher);
+        } else if (newLevel >= this.getMaxBuildingLevel()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this "else" keyword to a new dedicated line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ARightCurlyBraceDifferentLineAsNextBlockCheck)